apps/docs/src/.vitepress/components/ComponentCard.vue

```diff
@@ -1,14 +1,33 @@
 <script lang="ts" setup>
+import { computed } from "vue";
+import OnyxHeadline from "~components/OnyxHeadline/OnyxHeadline.vue";
+import type { ComponentStatus } from "./ComponentStatusBadge.vue";
+import ComponentStatusBadge from "./ComponentStatusBadge.vue";
+
 export type ComponentCardProps = {
   /** Component name. */
   name: string;
-  /** If true an "Implemented" badge will be shown, "Planned" otherwise. */
-  implemented?: boolean;
-  /** Link to the implemented component. */
+  /** Component status. */
+  status: ComponentStatus;
+  /** Link to the component. */
   href?: string;
+  /**
+   * Estimation date when the component will be implemented.
+   * Only the month and year of the date will be shown.
+   * If unset, "n/a" will be displayed.
+   */
+  estimation?: ConstructorParameters<typeof Date>[0];
 };
 
 const props = defineProps<ComponentCardProps>();
+
+const estimationValue = computed(() => {
+  if (!props.estimation) return "n/a";
+
+  const date = new Date(props.estimation);
+  const month = (date.getMonth() + 1).toString().padStart(2, "0");
+  return `${month}/${date.getFullYear()}`;
```

Comment:
Prefer the [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) for formatting of dates

Reply:
Done :)